crypter.go

```diff
@@ -2,7 +2,7 @@ package walg
 
 import "io"
 
-// Crypter is responsible for makeing cryptographical pipeline parts when needed
+// Crypter is responsible for making cryptographical pipeline parts when needed
 type Crypter interface {
```

Comment:
Currently, we have compressor and decompressor. How do you think, is it viable to split crypter into encryptor and decryptor? They share keyring infrastructure and we have an issue about them #92, maybe keepeing them together for now is a good idea.